docker/ubuntu1604/Dockerfile

```diff
@@ -52,8 +52,9 @@ RUN echo "deb http://apt.openvstorage.com unstable main" > /etc/apt/sources.list
     	    libxio0 libxio0-dbg libxio0-dbgsym \
     	    libxio-dev libxio-dev-dbgsym && \
     apt-get install -y libev4 && \
-    cd /root && apt-get download --allow-unauthenticated arakoon && \
+    cd /tmp && apt-get download --allow-unauthenticated arakoon && \
```

Comment:
/tmp is shared between all builds on the internal build system, so better not use it. The downloaded deb is removed a couple of lines further, so nothing is left behind.